### PR TITLE
cookieDomain must be correct

### PR DESCRIPTION
**Pull request by adrai**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -22,12 +22,13 @@ i18next
...
       addPath: __dirname + '/locales/{{lng}}/{{ns}}.missing.json'
     },
+    preload: ['en', 'es'],
     fallbackLng: 'en',
     saveMissing: true,
     detection: {
       // Use cookies so the user selected language is kept
       caches: ['cookie'],
-      cookieDomain: 'i18next-express-middleware-test'
+      cookieDomain: 'localhost' // => must be a real domain
```
Comment (alvarofrutos) [inline app.js]: True, this is solving the issue with the cookie. Thanks.

File: app.js [hunk 2]
```diff
@@ -44,9 +45,7 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Use i18next
-app.use(i18midd.handle(i18next, {
-  removeLngFromUrl: false
-}));
+app.use(i18midd.handle(i18next));
```
Comment (alvarofrutos) [inline app.js]: Are you suggesting this for some reason? The issue is solved just with the `cookieDomain` being correctly set.

File: app.js [hunk 3]
```diff
@@ -22,12 +22,13 @@ i18next
       loadPath: __dirname + '/locales/{{lng}}/{{ns}}.json',
       addPath: __dirname + '/locales/{{lng}}/{{ns}}.missing.json'
     },
+    preload: ['en', 'es'],
```
Comment (alvarofrutos) [inline app.js]: Are you suggesting this for some reason? The issue is solved just with the `cookieDomain` being correctly set.

Comment (adrai) [reply app.js]: image
https://www.i18next.com/overview/configuration-options

File: app.js (hunk 2 above)
Comment (adrai) [reply app.js]: false is default: https://github.com/i18next/i18next-express-middleware/blob/013af5ed30410bb5375375c7aa1fe6219c2b783a/src/index.js#L52

**Pull request merged**